code/game/objects/items/rcd/RSF.dm

```diff
@@ -32,10 +32,16 @@ RSF
 	w_class = WEIGHT_CLASS_NORMAL
 	///An associated list of atoms and charge costs. This can contain a separate list, as long as it's associated item is an object
 	var/list/cost_by_item = list(/obj/item/reagent_containers/cup/glass/drinkingglass = 20,
+								/obj/item/reagent_containers/cup/glass/sillycup = 10,
 								/obj/item/paper = 10,
 								/obj/item/storage/dice = 200,
 								/obj/item/pen = 50,
 								/obj/item/clothing/mask/cigarette = 10,
+								/obj/item/plate = 70,
+								/obj/item/reagent_containers/cup/bowl = 70,
+								/obj/item/kitchen/fork/plastic = 30,
+								/obj/item/knife/plastic = 30,
+								/obj/item/kitchen/spoon/plastic = 30,
 								)
```

Comment:
this is a really bad list multiline and it's not your fault, but let's fix it.

```suggestion
	var/list/cost_by_item = list(
		/obj/item/reagent_containers/cup/glass/drinkingglass = 20,
		/obj/item/reagent_containers/cup/glass/sillycup = 10,
		/obj/item/paper = 10,
		/obj/item/storage/dice = 200,
		/obj/item/pen = 50,
		/obj/item/clothing/mask/cigarette = 10,
		/obj/item/plate = 70,
		/obj/item/reagent_containers/cup/bowl = 70,
		/obj/item/kitchen/fork/plastic = 30,
		/obj/item/knife/plastic = 30,
		/obj/item/kitchen/spoon/plastic = 30,
	)
```

it would also be great if you could alphabetize this (via a vscode extension) or arrange it by weights or whatever.

Reply:
Honestly the way it's arranged right now is by use, like the glass and paper cup together, the paper, dice and pen together, cigarette... in there where it usually was, the plate and bowl together, and the plastic cutlery together. 
If you'd still prefer it being alphabetical or by weights or somesuch, do dell

Reply:
does it really show up in this exact order in some player-facing UI? If so, put a comment delineating that and i'm happy because otherwise someone in the not-so-distant future will see this and go "silly coder didn't alphabetize!" and screw it all up.

should defo still be multi-lined though.

Reply:
Yeah it shows up in this order in the playerfacing menu when using the RSF,  the same with the borgshaker and condiment synthesizer, the things written first shows first, the next comes after, etc etc.

Reply:
So anyway I left a comment there specifying why it isn't alphabetical and did the multi-line thing. At least I think that's what you meant by multi line.